shared/src/main/scala/com/cibo/evilplot/plot/components/GenericDrawable.scala

```diff
@@ -0,0 +1,33 @@
+package com.cibo.evilplot.plot.components
+
+import com.cibo.evilplot.geometry.{Drawable, Extent}
+import com.cibo.evilplot.plot.Plot
+import com.cibo.evilplot.plot.aesthetics.Theme
+
+case class GenericDrawable(
```

Comment:
I think we should make this `final`, I also think `GenericDrawable` is perhaps a misleading name for this, since it strongly evokes `Drawable`

Reply:
Maybe change name to Marker? That way it lines up with its description in the examples and in EDA.